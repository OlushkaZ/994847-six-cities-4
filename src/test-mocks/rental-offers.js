export const RENTAL_OFFERS = [
  {
    id: 1,
    location: {
      cityCoordinates: [52.3909553943508, 4.85309666406198],
    },
    img: `img/apartment-01.jpg`,
    price: 100,
    rating: `95%`,
    name: `Beautiful luxurious apartment at great location`,
    type: `hostel`,
    isPremium: true,
    isBookmark: false,
  },

  {
    id: 2,
    location: {
      cityCoordinates: [52.3909553943508, 4.85309666406198],
    },
    img: `img/apartment-02.jpg`,
    price: 193,
    rating: `60%`,
    name: `Canal View Prinsengracht`,
    type: `apartment`,
    isPremium: true,
    isBookmark: true,
  },

  {
    id: 3,
    location: {
      cityCoordinates: [52.3909553943508, 4.85309666406198],
    },
    img: `img/apartment-03.jpg`,
    price: 250,
    rating: `16%`,
    name: `Nice, cozy, warm big bed apartment`,
    type: `hotel`,
    isPremium: false,
    isBookmark: false,
  },

  {
    id: 4,
    location: {
      cityCoordinates: [52.3909553943508, 4.85309666406198],
    },
    img: `img/apartment-01.jpg`,
    price: 50,
    rating: `75%`,
    name: `Wood and stone place`,
    type: `house`,
    isPremium: false,
    isBookmark: true
  }
];

export const RENTAL_OFFER = {
  id: 1,
  img: `img/apartment-01.jpg`,
  price: 100,
  rating: `95%`,
  name: `Beautiful luxurious apartment at great location`,
  type: `hostel`,
  isPremium: true,
  isBookmark: false
};

