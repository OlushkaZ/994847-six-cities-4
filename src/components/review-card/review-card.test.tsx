import React from 'react';
import renderer from 'react-test-renderer';

import {REVIEWS} from '../../test-mocks/reviews';
import ReviewCard from './review-card';

it(`Should render RentalCard correctly`, () => {
  const tree = renderer.create((
    <ReviewCard review={REVIEWS[0]} />
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
