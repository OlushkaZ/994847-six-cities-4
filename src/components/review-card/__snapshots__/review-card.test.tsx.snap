// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render RentalCard correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="/img/avatar.svg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Michael
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": 3,
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      The proximity to the airport is great, very friendly owner, clean and tidy, parking right outside the door - a very good choice with an early onward journey to the airport with excellent value for money.
    </p>
    <time
      className="reviews__time"
      dateTime="2007-05-01"
    >
      May 2007
    </time>
  </div>
</li>
`;
